.vscode/settings.json

```diff
@@ -0,0 +1,39 @@
+{
+    "files.associations": {
```

Comment:
won't this propagate to vscode editing of dotnet-monitor outside the dev container scenario, or is that the intent?